plugins/outputs/stomp/README.md

```diff
@@ -0,0 +1,39 @@
+# STOMP Producer Output Plugin
+
+This plugin writes to a [Active MQ Broker](http://activemq.apache.org/) Applying STOMP Protocol.
+
+It also support Amazon MQ  (https://aws.amazon.com/amazon-mq/)
```

Comment:
```suggestion
This plugin sends metrics to an [Active MQ Broker](http://activemq.apache.org/) applying the STOMP protocol.
It also support Amazon MQ  (https://aws.amazon.com/amazon-mq/)
```
Do you have a reference for the `STOMP` protocol? It would be nice to link to it here...